ray-operator/main.go

```diff
@@ -117,15 +119,29 @@ func main() {
 		setupLog.Info("Feature flag enable-batch-scheduler is enabled.")
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	watchNamespaces := strings.Split(watchNamespace, ",")
+	options := ctrl.Options{
 		Scheme:                 scheme,
 		MetricsBindAddress:     metricsAddr,
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "ray-operator-leader",
-		Namespace:              watchNamespace,
-	})
+	}
+
+	if len(watchNamespaces) == 1 {
```

Comment:
It is not possible for `len(watchNamespaces) == 0` to be true. The length of `strings.Split("", ",")` is still 1. See [this link](https://stackoverflow.com/questions/28330908/how-to-string-split-an-empty-string-in-go) for more details.

Reply:
This seems worth including as a code comment.

Reply:
Added https://github.com/ray-project/kuberay/pull/1106/commits/f209400cc32087dfe5488f0b2e69b3c251ae4e8d.